packages/app/src/display/style/components/_lobby.scss

```diff
@@ -1,100 +1,112 @@
 .player {
-    padding: 0.4em 1em;
```

Comment:
just for future, these formatting changes (with the indentation) make it harder to review what's actually changed here

check the diff and avoid making changes like this (except in their own PRs 😄)